### DISCUSSION TITLE
Major TOM Dataset Wishlist

### DISCUSSION THREAD

**Discussion by mikonvergence** [ESA-PhiLab/Major-TOM#7] [Ideas]
This is a place where you can express interest and ideas for potential Major TOM expansions.

Feel free to react to other people's ideas and coordinate on building future expansions to Major TOM.

This includes both:
* New data sources
* New sampling of exisitng sources
* New labels

**Comment by LewChen** [comment]
Hello, I wanna to use MajorTom-Core Dataset to do cloud detection based on deep learning, But I don't know how to download this dataset that I need. Is there someone who can help me ?

**Reply by mikonvergence** [reply]
[CloudSEN12](https://cloudsen12.github.io/) might be what you are looking for. It is not as dense as Major TOM, but it has very likely the highest quality of labels. Is there a specific reason why you would like to see Major TOM expansion for cloud masking?
